src/main/resources/io/smallrye/coderules/eclipse-format.xml

```diff
@@ -0,0 +1,348 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It's hard to tell as the list isn't ordered. Is this exactly the same as what Quarkus currently has or were there changes?

Reply:
I think the only change I made was the line length. It was changed from 120 to 140 chars. With normal screen size this should be fine.